client/dashboard/.eslintrc.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+	rules: {
+		'no-restricted-imports': [
+			'error',
+			{
+				patterns: [
+					{
+						group: [
+							'calypso/*',
+							// Allowed: calypso/lib/wp
+							'!calypso/lib',
+							'calypso/lib/*',
```

Comment:
Are the above two lines necessary? (similar question for the rest of the file), do we need to do this nesting thing or can we just allow the specific parts directly.

Reply:
weird

Reply:
As discussed on Slack, it doesn't work